test/encode.js

```diff
@@ -31,11 +31,24 @@ describe('encode', function() {
     expect(uri).to.equal('data:text/plain;base64,Zm9v');
   });
 
+  it('creates from buffer with default mediatype', function() {
+    var uri = dataUri.encode(new Buffer('foo', 'utf8'));
+    expect(uri).to.be.a('string');
+    expect(uri).to.equal('data:application/octet-stream;base64,Zm9v');
+  });
+
   it('creates from string', function() {
     // default mediatype
```

Comment:
A forgotten comment? I think it should be removed, because this test is setting an explicit mediatype "text/xml".